Proposal for 🐛 Bug Report Issue Form

VoxelMC opened the pull request:
### Description

Adds an issue template using GitHub's beta forms feature.
This is an effort to standardize issue reports with the other main repos such as `astro` and `docs`.

My apologies if it's terrible, its my first issue template! I would love to implement feedback! 

I will also draft one up for feature requests.

Here is the rendered form:
![preview](https://github.com/withastro/houston-vscode/assets/20650404/f0d8344b-2bce-4904-9909-8d39b170a2b3)

VoxelMC committed:
Create Bug Report Issue Template

VoxelMC committed:
Update BUGS.yml

delucis commented on .github/ISSUE_TEMPLATE/BUGS.yml:
```diff
@@ -0,0 +1,56 @@
+      label: "❓ Issue Type"
+      description: Is this a visual issue or a problem with the extension? If `other`, please specify in the description.
+      options:
+        - Visual
+        - Extension
+        - Other
```
I wonder if “Visual” vs “Extension” is a clear distinction? I kind of feel like all bugs would be a problem with the extension. And indeed that these aren’t mutually exclusive — maybe the extension has a visual problem!

What do you think we gain from a dropdown like this? Maybe the form is just as helpful without it?

VoxelMC replied:
I think my goal was to triage between issues with the function of the extension (Houston disappears after x thing), and issues with things like colours and the scroll bar covering up the git history, etc. However, I see now that these are somewhat the same things - as anything wrong with the extension itself would be a visual problem in the end. 

I think I'll just remove it!

delucis replied:
Sounds good 👍
Thanks @VoxelMC!

delucis reviewed (commented):
Looks fantastic @VoxelMC — thank you!

Left one question for you.

VoxelMC committed:
Update BUGS.yml

VoxelMC commented:
@delucis Removed the dropdown, and changed my little Houston snippet. Updated the preview image in PR description, but here it is anyways: 

![image](https://github.com/withastro/houston-vscode/assets/20650404/2534ee28-3ceb-4343-ba19-c20b53ee0323)

delucis commented on .github/ISSUE_TEMPLATE/BUGS.yml:
````diff
@@ -0,0 +1,43 @@
+    attributes:
+      value: |
+        ```
+        ╭─────╮    > Thank you for reporting your issue! 
+        │ ◠ ◡ ◠
+        ╰─────╯
````
```suggestion
        ╭─────╮
        │ ◠ ◡ ◠   Thank you for reporting your issue! 
        ╰─────╯
```

delucis reviewed (approved):
This is lovely @VoxelMC! Going to make one small tweak to the text positioning in the Houston snippet and then get this merged. Thank you again! 🙌

delucis committed:
Tweak Houston ASCII art
